docs/src/rest/partition-spec.md

```diff
@@ -0,0 +1,161 @@
+## Lance Partition Spec (Experimental)
+
+### 1. Overview
+This specification defines a standard for describing and operating on **Partitioned Tables** at the **Lance Namespace** level. A partitioned table is a logical table whose data is physically split and stored in multiple independent Lance `Table` objects.
+
+The core objective of this specification is to provide a standardized and efficient method for organizing, managing, and accessing large-scale datasets without depending on a specific computation engine.
+
+This specification follows the "convention over configuration" principle, defining a partitioned table as a **metadata specification**. It is the responsibility of compatible clients or computation engines to interpret and enforce it.
+
+### 2. Core Concepts
+#### 2.1. Partitioned Table
```

Comment:
I think calling this a partitioned table would be quite confusing to many people because the actual interface is namespace not table. What about "Partitioned Namespace"?